lib/fluent/plugin/out_influxdb.rb

```diff
@@ -106,6 +114,10 @@ def shutdown
     @influxdb.stop!
   end
 
+  def formatted_to_msgpack_binary
+    true
+  end
+
   def write(chunk)
     points = []
     chunk.msgpack_each do |tag, time, record|
```

Comment:
Can we reuse `chunk.metadata.tag`?
If so, I can change format routine to `[precision_time(time), record].to_msgpack` instead of `[tag, precision_time(time), record].to_msgpack`.
How about this?